commons/src/main/java/io/hyscale/commons/constants/ValidationConstants.java

```diff
@@ -64,5 +64,6 @@
     public static final String STRUCTURED_OUTPUT_FORMAT_REGEX = "JSON|json";
 
     public static final String INVALID_OUTPUT_FORMAT_MSG = "Output format \"{}\" is invalid. Supported format is json.";
-    
+
+    private ValidationConstants(){}
```

Comment:
Beginning

Reply:
In All Constants file I kept the constructors at the end as per coding conventions.